### ISSUE TITLE
Swagger-UI. Add request body example values.

### ISSUE THREAD

**Issue by lganzzzo** [oatpp/oatpp#368]
@oben01
Description copied from - #367 

I just get more information to set array example data.
A working example could be : 
```yaml
swagger: "2.0"

info:
  title: A dummy title
  version: 1.0.0

paths:
  /path:
    post:
      parameters:
        - in: body
          description: xxx
          required: true
          name: a name
          schema:
            type: array
            items:
              type: string
            example: ["str1", "str2", "str3"]
      responses:
        default:
          description: OK
```

That will produce an array of ["str1", "str2", "str3"]

![github](https://user-images.githubusercontent.com/22369027/103443905-d0979380-4c63-11eb-8a05-a29c8d955e04.PNG)

Is that possible to add this data with the current API ? if not could you please add it ??

**Comment by oben01** [comment]
Hello @lganzzzo,

I defined my DTO_FIELD vector with default value as :
```c++
class KeywordsListInfoBody : public oatpp::DTO
{
   DTO_INIT(KeywordsListInfoBody, DTO);
   DTO_FIELD(Vector<String>, keywords);
   DTO_FIELD(Vector<String>, info) = Vector<String>({"lastValue","lastValueDate","exist"});
 
};
```
I can see my default values only in Schema
![schema](https://user-images.githubusercontent.com/22369027/103460539-3bee6d80-4d17-11eb-9684-d0236e3419f9.PNG)
And the example still not getting this default values set 

![RequestValue](https://user-images.githubusercontent.com/22369027/103460545-4d377a00-4d17-11eb-97f8-756de73c209b.PNG)

I copied the swagger json contract from api-docs/oas-3.0.0.json to swagger editor, magically I get my default values set in the body 

![swaggereditor](https://user-images.githubusercontent.com/22369027/103460601-c505a480-4d17-11eb-9cb2-dfeac9efbc2d.PNG)

I don't understand this delta between swagger editor and oatp++ swagger page ....
If you got any idea please let me know.
Thank you so much 👍 

the generated swagger yml (Keyword List info endpoint):

```yml
openapi: 3.0.0
info:
  title: Yadoms server Swagger-UI
  description: 'Yadoms is open source, simple, powerfull, flexible and multiplatforms domotic solution.'
  contact:
    name: Oussama DAHMAZ
    url: 'http://yadoms.com/forum/'
  license:
    name: GNU General Public License v3.0
    url: 'https://github.com/Yadoms/yadoms-swagger-spot/blob/master/LICENSE'
  version: '1.0'
servers:
  - url: 'http://localhost:8080/rest'
    description: server on localhost
paths:
  /acquisition/keyword/lastdata:
    put:
      description: Get the last acquisition for a list of keywords passed as a body
      operationId: getKeywordListLastData
      requestBody:
        description: request body
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/KeywordsLastDataBody'
      tags:
        - Keyword List Last data
      responses:
        '200':
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/KeywordsLastDataResponse'
      parameters: []
  /acquisition/keyword/info:
    put:
      description: Get the last acquisition for a list of keywords passed as a parameter
      operationId: getKeywordListInfo
      requestBody:
        description: request body
        content:
          application/json:
            schema:
              $ref: '#/components/schemas/KeywordsListInfoBody'
      tags:
        - Keyword List info
      responses:
        '200':
          description: OK
          content:
            application/json:
              schema:
                $ref: '#/components/schemas/KeywordListInfoResponse'
      parameters: []
components:
  schemas:
    LastValue:
      type: object
      properties:
        lastValue:
          type: string
    KeywordsLastDataResponse:
      type: object
      properties:
        result:
          type: boolean
        message:
          type: string
        data:
          type: object
          additionalProperties:
            $ref: '#/components/schemas/LastValue'
    KeywordsLastDataBody:
      type: object
      properties:
        keywords:
          type: array
          items:
            type: string
    Info:
      type: object
      properties:
        lastValue:
          type: string
        lastValueDate:
          type: string
        exist:
          type: string
    KeywordListInfoResponse:
      type: object
      properties:
        result:
          type: boolean
        message:
          type: string
        data:
          type: object
          additionalProperties:
            $ref: '#/components/schemas/Info'
    KeywordsListInfoBody:
      type: object
      properties:
        keywords:
          type: array
          items:
            type: string
        info:
          type: array
          default:
            - lastValue
            - lastValueDate
            - exist
          items:
            type: string

```

**Comment by lganzzzo** [comment]
Hey @oben01 ,

> I don't understand this delta between swagger editor and oatp++ swagger page ....
If you got any idea please let me know.

Please get the latest master from `oatpp-swagger`. I've updated swagger-UI to the latest available.
The default values will be shown as you've described.

Regards,
Leonid

**Comment by oben01** [comment]
Hi @lganzzzo 
I confirm that the default values are shoween 
Thank you

**Comment by lganzzzo** [comment]
Hey @oben01 ,

I've checked with Open-API spec and your example is an invalid swagger spec:

```yaml
swagger: "2.0"

info:
  title: A dummy title
  version: 1.0.0

paths:
  /path:
    post:
      parameters:
        - in: body # <--- invalid value 'body' !!!
          description: xxx
          required: true
          name: a name
          schema:
            type: array
            items:
              type: string
            example: ["str1", "str2", "str3"]
      responses:
        default:
          description: OK
```

**Note:** There is no `body` for the field `in` - see [Describing Parameters](https://swagger.io/docs/specification/describing-parameters/)

Instead, you have to use `requestBody` section - see [Describing Request Body](https://swagger.io/docs/specification/describing-request-body/)

**Comment by lganzzzo** [comment]
@oben01 

Request body example functionality is now available - please get the latest `oatpp` and `oatpp-swagger`.
Please also note that this functionality is not final, and will be modified to support multiple examples per entity.

At the moment you can use the following code:

```cpp
  ENDPOINT_INFO(myEndpoint) {
    info->body.example = <any value>; // example oatpp::Vector<oatpp::String>({"a", "b", "c"});
  }
```

This functionality is provided for "play-with" purposes and will be changed shortly to:

```cpp
  ENDPOINT_INFO(myEndpoint) {
    info->body.addExample("example-1", oatpp::Vector<oatpp::String>({"a", "b", "c"}));
  }
```

**Comment by oben01** [comment]
Hello @lganzzzo 
I confirm that description is set correctly
Thank you again 🗡️

**Comment by lganzzzo** [comment]
Implemented in `1.2.5` see the changelog for details - [1.2.5 changelog](https://github.com/oatpp/oatpp/blob/master/changelog/1.2.5.md#swagger-ui-example-values)
